src/client.js

```diff
@@ -41,6 +42,13 @@ class Client {
     this.currentCabal = null
     this.config = opts.config
     this.maxFeeds = opts.maxFeeds || 1000
+    this.aliases = opts.aliases || {}
+    this.commands = Object.assign({}, defaultCommands, opts.commands)
```

Comment:
ahh nice - this should e.g. allow for cabal-desktop to overload any default commands, right?

Reply:
yes. you can also use client.{add,remove}Command() and client.{add,remove}Alias() methods

Reply:
cc @nikolaiwarner good to know for desktop ^ :3